lib/plottr_components/src/components/dashboard/files/FileActions.js

```diff
@@ -108,24 +120,28 @@ const FileActionsConnector = (connector) => {
     return (
       <div className="dashboard__recent-files__file-actions">
         {renderDeleteFile()}
-        <Dropdown id={`file-action-${id}`} onSelect={doTheThing}>
-          <Dropdown.Toggle noCaret>
-            <Glyphicon glyph="option-horizontal" />
-          </Dropdown.Toggle>
-          <Dropdown.Menu>
-            {missing ? null : <MenuItem eventKey="open">{t('Open')}</MenuItem>}
-            {isCloudFile || osIsUnknown || missing ? null : (
-              <MenuItem eventKey="show">{showInMessage}</MenuItem>
-            )}
-            {missing || offline ? null : <MenuItem eventKey="rename">{t('Rename')}</MenuItem>}
-            {(isCloudFile && permission !== 'owner') || missing ? null : (
-              <MenuItem eventKey="delete">{t('Delete')}</MenuItem>
-            )}
-            {(isTemp || missing) && !osIsUnknown && (
+        <ButtonGroup>
+          <Button bsSize="small" onClick={handleOpen}>
```

Comment:
If the file's missing, we shouldn't show any of these three buttons, or they should be disabled.

Reply:
Good catch @Quiescent thank you for checking :bow: